### DISCUSSION TITLE
PHPRedis upgrade to 6.0 in debian bullseye

### DISCUSSION THREAD

**Discussion by tonnyb** [oerdnj/deb.sury.org#2032] [Q&A]
Hi,

On 12 September 2023 phpredis was upgraded from 5.3.X to 6 for PHP 8.1+ on debian bullseye and there was no php redis 5.3.x candidate for php 8.1, i was wondering what the idea was behind this change, because the upgrade did bring some big changes, maybe i do know the answer but i'd be more than happy to know why it was chosen not to include 5.3.x

**Comment by oerdnj** [comment]
You are using repository that uses latest upstream versions. You need to decide whether you want the latest version and yesterday was too late or only when you like that...
